src/util/std_types.h

```diff
@@ -1087,33 +1000,27 @@ class incomplete_array_typet:public type_with_subtypet
   }
 };
 
-/*! \brief Cast a generic typet to an \ref incomplete_array_typet
- *
- * This is an unchecked conversion. \a type must be known to be \ref
- * incomplete_array_typet.
- *
- * \param type Source type
- * \return Object of type \ref incomplete_array_typet
- *
- * \ingroup gr_std_types
-*/
+/// \brief Cast a generic typet to an \ref incomplete_array_typet
+///
+/// This is an unchecked conversion. \param type must be known to be \ref
+/// incomplete_array_typet.
+///
+/// \param type Source type.
+/// \return Object of type \ref incomplete_array_typet.
 inline const incomplete_array_typet &to_incomplete_array_type(const typet &type)
 {
   PRECONDITION(type.id()==ID_array);
   return static_cast<const incomplete_array_typet &>(type);
 }
 
-/*! \copydoc to_incomplete_array_type(const typet &)
- * \ingroup gr_std_types
-*/
+/// \copydoc to_incomplete_array_type(const typet &)
 inline incomplete_array_typet &to_incomplete_array_type(typet &type)
 {
   PRECONDITION(type.id()==ID_array);
   return static_cast<incomplete_array_typet &>(type);
 }
 
-/*! \brief Base class of bitvector types
-*/
+/// Base class of bitvector types
```

Comment:
Give an example of a bitvector type as it isn't obvious these are integers for example